docs/modelserving/nodescheduling/inferenceservicenodescheduling.md

```diff
@@ -0,0 +1,269 @@
+# InferenceService Node Scheduling
+## Setup
+The `InferenceService` spec supports node selector, node affinity and tolerations. To enable these features we must enable the knative flags (see [Install Knative Serving Note](https://kserve.github.io/website/admin/serverless/#2-install-knative-serving)).
+
+### Option 1: Pre-Kubeflow Install Feature Flags Setup
+If we install KServe as part of Kubeflow manifest and would like to enable the feature flags before installing Kubeflow, we can do so by editing the file `manifests/common/knative/knative-serving/base/upstream/serving-core.yaml`   
+This is often a common approach that allows a reproducible configuration as the feature flags will be enabled everytime we install Kubeflow.  
+
+
+1. Enable kubernetes.podspec-affinity  
+```yaml
+kubernetes.podspec-affinity: "enabled"
+```
+2. Enable kubernetes.podspec-nodeselector  
+```yaml
+kubernetes.podspec-nodeselector: "enabled"
+```
+3. Enable kubernetes.podspec-tolerations  
+```yaml
+kubernetes.podspec-tolerations: "enabled"
+```
+
+With all features enabled we should have a `data` portion that looks like this : 
+```yaml
+data:
+  kubernetes.podspec-affinity: "enabled"
+  kubernetes.podspec-nodeselector: "enabled"
+  kubernetes.podspec-tolerations: "enabled"
+```
+
+### Option 2: Post-Kubeflow Install Feature Flags Setup  
```

Comment:
We can also list a 3rd option for KServe standalone installation but I do not know where the config file is for the standalone install. Some help would be appreciated.